integration/src/integration/java/pl/allegro/tech/hermes/integration/AsyncPublisherAutoRetryTest.java

```diff
@@ -53,8 +57,9 @@ public void shouldRetryPublishingMessageEnoughTimesWhenServerReturns500() {
     }
 
     @Test
-    public void shouldNotRetryIfMessageWasPublished() {
-        HermesClient hermesClient = retryingHermesClient(10);
+    public void shouldNotRetryIfMessageWasPublishedUsingRestTemplate() {
```

Comment:
Nit: as this test uses `AsyncRestTemplate`, maybe it should be named `shouldNotRetryIfMessageWasPublishedUsingAsyncRestTemplate`?